--- a/makemkvgui/src/mainwnd.cpp	2026-10-01 22:25:01.571486975 +0200
+++ b/makemkvgui/src/mainwnd.cpp	2026-10-01 22:25:01.584847922 +0200
@@ -1282,7 +1282,12 @@
         {
             real_url = QLatin1String("http://127.0.0.1/?hide?") + url;
         }
+#if 0
         QDesktopServices::openUrl(QUrl(real_url));
+#else
+        QMessageBox::information(NULL,UI_QSTRING(APP_CAPTION_MSG),
+            QLatin1String("Please visit the following URL: ")+real_url);
+#endif
     }
     m_disable_onidle--;
 }
